lib/hackerone/client/report.rb

```diff
@@ -75,6 +75,32 @@ def activities
         end
       end
 
+      def program
+        @program || Program.find(relationships[:program][:data][:attributes][:handle])
+      end
+
+      def assign_to_user(name)
+        fail(
```

Comment:
Hey! Sorry for the late reply.

I see your point, and I think it's a fair suggestion. I don't think, however, it's helpful to generate error messages after the fact. If we choose to be a "dump" API so to say, and stay relatively close to the metal, I think we should stick with that and not try to be clever like I'm doing in this PR. 

Moreover, I find it a bit silly to generate helpful error messages after doing a request. After all, if you're able to determine that the request is going to fail (and why), how come you even allowed me to make the request? That seems odd to me.